blueflood-core/src/main/java/com/rackspacecloud/blueflood/io/IMetricsWriter.java

```diff
@@ -1,28 +0,0 @@
-/*
```

Comment:
This interface is no longer needed. Instead we use the encapsulation of *MetricsRW classes.